TFT/src/User/API/HomeOffsetControl.c

```diff
@@ -3,7 +3,6 @@
 
 static float z_offset_value = HOME_Z_OFFSET_DEFAULT_VALUE;
 static bool home_offset_enabled = false;
-
```

Comment:
I suspect you removed the empty line by mistake

Reply:
some old habits